### ISSUE TITLE
Rails 5.1 Support

### ISSUE THREAD

**Issue by sgringwe** [Mange/roadie-rails#68]
When upgrading to rails 5.1, roadie-rails has an incompatibility:

```
roadie-rails was resolved to 1.0.0, which depends on
      rails (< 4.2, >= 3.0)
```

**Comment by shakaman** [comment]
You can check #69

**Comment by glebm** [comment]
#70 also resolves it. @Mange This is affecting a lot of users. Any chance you might be able to release a new version today?

**Comment by Mange** [comment]
Fixed in #70 with version 1.2.0 released.
